symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -240,6 +253,21 @@ protected SSLContext createSSLContext() {
       sslConfig
           .trustStoreBytes(trustStoreBytes)
           .trustStorePassword(trustStorePassword);
+      // if logging debug is enabled, we print the truststore entries
+      if (logger.isDebugEnabled()) {
+        final KeyStore truststore;
+        try {
+          truststore = KeyStore.getInstance(TRUSTSTORE_FORMAT);
+          truststore.load(new ByteArrayInputStream(trustStoreBytes), trustStorePassword.toCharArray());
+          final List<String> aliases = Collections.list(truststore.aliases());
+          logger.debug("Your custom truststore contains {} entries :", aliases.size());
+          for (String alias : aliases) {
+            logger.debug("# {}", alias);
+          }
+        } catch (KeyStoreException | NoSuchAlgorithmException | CertificateException | IOException e) {
+          e.printStackTrace();
```

Comment:
We usually never want to use printStackTrace because it uses stderr instead of a logger so we don't have control over where the logs ended up

Reply:
I will replace it by throwing the runtime exception, in next commit.